src/build.ts

```diff
@@ -184,7 +184,8 @@ async function optimizePng(filePath: string, png: PngOptions) {
   }
 }
 
-function resolveTempPngAssetName(name: string) {
+async function resolveTempPngAssetName(name: string) {
+  await mkdirSync(dirname(name), { recursive: true })
```

Comment:
use `mkdir` here

Reply:
It was my negligence, it has been fixed now